kube-enforcer/README.md

```diff
@@ -71,21 +81,25 @@ Optional flags:
 --aquaSecret.kubeEnforcerToken           default to "" you can find the KubeEnforcer token from aqua csp under enforcers tab in default/custom KubeEnforcer group or you can manually approve KubeEnforcer authentication from aqua CSP under default/custom KubeEnforcer group in enforcers tab.
 ```
 
+## ClusterRole
+
+KubeEnforcer needs a dedicated clusterrole with **get, list, watch** permissions on **pods, secrets, nodes, namespaces, deployments, replicasets, replicationcontrollers, statefulsets, daemonsets, jobs, cronjobs, clusterroles, clusterrolebindings, componentstatuses** to perform discovery on the cluster.  
```

Comment:
This Chart already contains a `ClusterRole`, I would find this very confusing; this makes it sound like I need to add a `ClusterRole` myself.

I've also made a bunch of PRs to fix some very confusing and buggy inconsistencies and this adds a new inconsistency, the Server Chart also has [a `ClusterRole`](https://github.com/aquasecurity/aqua-helm/blob/7ec21a0f3f4828c82dc1df20cfa97dcc8faafbc2/server/templates/rbac.yaml#L28), but nothing is mentioned there...

It also doesn't say why it needs each of those permissions.